tests/cli.rs

```diff
@@ -215,7 +215,7 @@ For more information try --help
 			"--pubkey",
 			"020000000000000000000000000000000000000000000000000000000000000000",
 		],
-		"Execution failed: invalid pubkey: Encoding(Secp256k1(InvalidPublicKey))\n",
+		"Execution failed: invalid pubkey: string error\n",
```

Comment:
In a548ad8050c5628d5596e6ebc33c387ae23f7e23:

Heh, this is unfortunate. It's ultimately a bug in rust-bitcoin's `ParsePublicKeyError` that its errors have a terrible `Display` impl. Nothing we can do here.